### PR TITLE
(Home) Performance

### PR DESCRIPTION
**Pull request by melcidar**
MAIN
 * this is just so I don't lose my progress
 * I have made charts using random data
 * The data in Number of projects, Total project value, etc., is still just added text, later I will make it so it's data
 * I have to add states and things to buttons, such as Home on side bar and 2023 performance, also drop down for year selection...
 * All in all, everything is just work in progress. 
 OTHER
 * ///// 
 
current look of the page:
image
image
image

### CODE REVIEW

File: src/components/home/Home.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                    }`}
+                    onClick={() => handleChange('Development & Revenue Costs')}
+                    >
+                    Development & Revenue Costs
+                </div>
+                <div
+                    className={`selection3 ${
+                        selectedOption === '2023 Plan' ? 'highlight' : ''
+                    }`}
+                    onClick={() => handleChange('2023 Plan')}
```
Comment (becarselman) [inline src/components/home/Home.js]: move this function inside of onClick outside of jsx

File: src/components/home/Home.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                    >
+                    2023 Performance
+                </div>
+                <div
+                    className={`selection2 ${
+                        selectedOption === 'Development & Revenue Costs'
+                        ? 'highlight'
+                        : ''
+                    }`}
+                    onClick={() => handleChange('Development & Revenue Costs')}
```
Comment (becarselman) [inline src/components/home/Home.js]: move this function inside onClick outside of jsx on all places

File: src/components/home/Home.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                    2023 Plan
+                </div>
+          </div>
+                <div className='year-selection'>
+                    <div className='year-text'>Year:</div>
+                    <Select defaultValue="2023" options={items} onChange={handleChange} />
+                </div>
+            </div>
+            <div className='cards'>
+                {data.map((item) => (
```
Comment (becarselman) [inline src/components/home/Home.js]: move map outside of jsx

File: src/components/home/Home.scss [hunk 4]
```diff
@@ -0,0 +1,249 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap');
+@import '../../utilis/colors';
+@import '../../utilis/mixins';
+
+.container{
+    display: flex;
+    flex-direction: row;
+    //height: 100vh;
```
Comment (becarselman) [inline src/components/home/Home.scss]: remove commented code

**Review by becarselman — APPROVED** [review]

**Review by ElvaRivle — APPROVED** [review]

**Pull request merged**